.circleci/config.yml

```
@@ -695,46 +713,98 @@ jobs:
         default: "14.3.1"
     macos:
       xcode: << parameters.xcode >>
+    resource_class: m4pro.medium
     environment:
       HOMEBREW_NO_AUTO_UPDATE: 1
-    resource_class: m4pro.medium
     steps:
       - checkout
       - *restore-cache-gems
-      - *restore-cache-podmaster
       - *install-gems
       - *prepare-netrc-file
       - *add-github-to-known-hosts
-      - install-mbx-ci
       - run:
           name: Generate docs
           command: |
-            OUTPUT=$( echo << pipeline.git.tag >> | sed 's/^v//' )
-            export OUTPUT="${OUTPUT:-documentation}"
-            echo "export OUTPUT=$OUTPUT" >> $BASH_ENV
-            ./scripts/document.sh
-            cp -a "$OUTPUT" /tmp/generated-docs
-      - *save-cache-podmaster
+            TAG="<< pipeline.git.tag >>"
+            if [ -n "$TAG" ]; then
+              VERSION=${TAG#v}
+            else
+              VERSION=0.0.0
+            fi
```

I'd suggest to fail fast instead of falling back with `0.0.0` to avoid silent continue:
```suggestion
            if [ -z "$TAG" ]; then
              echo "ERROR: No git tag found. Documentation generation requires a tagged release."
            exit 1
fi
```

---

In `generate-docs-job` job that is not triggered by a tag push (standard branch push) we still want to build docs for verification, so 0.0.0 is just a dummy version in this case. But in `publish-docs-job` jobs it fails fast in case there is no tag.